### PR TITLE
Sex Reagents!

### PR DESCRIPTION
**Pull request by Hmeister-real**
## About The Pull Request

Adds Cum as a fluid, adds the action to cum into containers, and adds milking breasts into containers. Also makes health potions taste like health rather than red, like manna potions tasting like manna

## Why It's Good For The Game

good for immersion

### CODE REVIEW

**Comment by Pikadoko** [comment]
Piss but no shit? This is a total beggar neglect, someone close this!

**Comment by Jaaackx** [comment]
PISSCODE IS FINALLY HERE.

**Comment by CalamaBanana** [comment]
Got milk?

**Comment by wow50000** [comment]
add milk

**Comment by DeerJesus** [comment]
pr finished. you can milk weenors and breasts. your own or others, forcibly. no piss stuff, yet, though the reagent is in the game for when we need it.

**Comment by wow50000** [comment]
force milk genitals only applies to males ?

**Comment by DeerJesus** [comment]
Yeah and I still think regular milk genitals should too but the calls up to
hmeister

**Comment by Hmeister-real** [comment]
> Yeah and I still think regular milk genitals should too but the calls up to hmeister

I mean you can milk a non-male, but it is kinda eh. if you want to make it male only you're free to do so

**Comment by wow50000** [comment]
add vagina milking you coward !

**Comment by kepesk1** [comment]
why is this immersive ew. no

**Comment by Coolgat3** [comment]
merge this already

File: code/modules/roguetown/roguejobs/alchemist/reagents.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	description = "Gradually regenerates all types of damage."
 	reagent_state = LIQUID
 	color = "#ff0000"
-	taste_description = "red"
+	taste_description = "health"
```
Comment (Cyprex) [inline code/modules/roguetown/roguejobs/alchemist/reagents.dm:6]: Good, but mention such minor changes on the sidelines

Comment (Hmeister-real) [reply code/modules/roguetown/roguejobs/alchemist/reagents.dm:6]: 👍

**Pull request merged**